### DISCUSSION TITLE
Adding confidence intervals

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1424] [Ideas]
Gró Einarsdóttir, Statistics Iceland:
>Hi there!

>First I wanted to congratulate you on your great SDG page! I'm so happy and grateful that you have developed it in an open source environment and that you are making it so easily accessible. We at Statistics Iceland are really interested in using your work for our reporting of the Icelandic SDG measurements. 

>I was trying to familiarize my self with the page and among other things read through the data scenarios. It made me wonder weather you have had any encounters with introducing confidence intervals into the graphs or whether you have any such plans for the future? It is especially important in Iceland to include confidence intervals, since we are such a small nations and our sample sizes are often not large enough to be certain that a given change is statistically significant. 
https://github.com/ONSdigital/sdg-indicators/wiki/Data-scenarios-and-characteristics

>Again, thank you for this great page and the easily understood instructions!

>Best

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @DarriEy We use the [Chart.js](https://www.chartjs.org/) library for our visualizations, and I'm wondering if this plugin might be relevant: https://github.com/datavisyn/chartjs-plugin-error-bars

I found that through this issue: https://github.com/chartjs/Chart.js/issues/6899

Do any of the visualizations there seem interesting? If so, which should we start with?

**Comment by EFT-Defra** [comment]
Any progress on this issue? I think we'd want to use the fill option mentioned in the Chart.js issue instead of the error bars provided by the plugin.
